### ISSUE TITLE
How to use pnp-js without the Import statements

### ISSUE THREAD

**Issue by rajivsr23** [pnp/pnpjs#701]
Thank you for reporting an issue, suggesting an enhancement, or asking a question. We appreciate your feedback - to help the team understand your
needs please complete the below template to ensure we have the details to help. Thanks!

**Please check out the [Docs](https://pnp.github.io/pnpjs/) to see if your question is already addressed there. This will help us ensure our documentation covers the most frequent questions.**

### Category
- [ ] Enhancement
- [ ] Bug
- [ x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [         ]

Please specify what version(s) of SharePoint you are targeting: [         ]

*If you are not using the latest release, please update and see if the issue is resolved before submitting an issue.*

### Expected / Desired Behavior / Question
With sp-pnp-js one could just use the require statements to Import the libraries. But with pnp-js it looks like the only way to do this is via Import statements? Is there a way to still use the require statements like we did with sp-pnp-js?

Basically I would like to use the following-
import { AdalFetchClient } from "@pnp/nodejs";
import { graph } from "@pnp/graph";

I am using Node js and Import statements are still not supported by default.

**Comment by koltyakov** [comment]
You can use `require` as well (```const { graph } = require('@pnp/graph');```) or ES6 modules and specific flags for Node.js, or, which is even better, Node.js + TypeScript which is industry standard and, IMO, a must have. ;)
